tests/test_dynamicbatching.py

```diff
@@ -124,6 +124,31 @@ def test_world_logging(self):
                         # we log the batch index in the teacher acts only
                         self.assertEquals(dyn_batch_idx, turn['dyn_batch_idx'])
 
+    def test_world_logging_buffersize(self):
+        """
+        Test world logging with dynamic batching when the number of
+        examples exceeds the buffersize.
+        """
+        with testing_utils.tempdir() as tmpdir:
+            save_report = os.path.join(tmpdir, 'report')
+            testing_utils.eval_model(
+                dict(
+                    model_file='zoo:unittest/transformer_generator2/model',
+                    task='integration_tests:RepeatTeacher:2000',
+                    world_logs=save_report,
```

Comment:
nit: `world_logs = save_report + '.jsonl'`

Reply:
Are you sure about that?

Reply:
https://github.com/facebookresearch/ParlAI/blob/master/tests/test_eval_model.py#L214-L215